openquake/commonlib/oqvalidation.py

```diff
@@ -220,6 +220,22 @@
 ebrisk_maxsize:
   INTERNAL
 
+floating_x_step:
+  Float, used in rupture generation for kite faults. indicates the fraction
```

Comment:
I find this a bit confusing. What about this: 
"Float, used in rupture generation for kite faults. Indicates the fraction of fault length used to float ruptures along strike (i.e. "0.5" floats the ruptures at half the rupture length)

Reply:
Okay - I've updated these descriptions and also added the explanation of what happens when the fraction is too small for the given combination of a rupture and a mesh